docs/concepts/zkAddress.md

```diff
@@ -2,7 +2,33 @@
 
 One essential concept in MantaPay is **zkAddresses**. A zkAddress is a reusable address for your private assets in Manta. It has the following properties:
 
-1. **Reusable**: A zkAddress can be safely used multiple times without compromising privacy by establishing correlations between transactions.
+1. **Reusable**: A zkAddress can be safely used multiple times without compromising privacy by establishing relations between transactions.
 2. **Public**: It is safe to publish your zkAddress without worrying about privacy leakage. MantaPay is designed such that zkAddresses are not disclosed on-chain.
-3. **Independent**: Your zkAddresses are not associated with any public addresses. For example, you can freely convert a public asset in public address `A` to private assets in shieled address `S`, and then convert it back to public assets in public address `B`.
-4. **Auditable**: All the transactions in a zkAddress can be audited using a [viewing key](ViewingKey.md).
+3. **Independent**: Your zkAddresses are not associated with any public addresses. For example, you can freely convert a public asset in public address `A` to private assets in a zkAddress `S`, and then convert it back to public assets in public address `B`.
+4. **Auditable**: All the transactions in a zkAddress can be audited using a viewing key.
+
+## zkAddress System
+
+The zkAddress is a part of our zkAddress system, which provides the user with different levels of rights over zkAssets.
+
+![zkAddress system](./resources/zkAddress.png)
+
+* The **zkAddress** grants the right to send zkAssets to the holder of the corresponding secret key.
+
+* The **viewing key** grants the right to view all transactions of the holder of the corresponding secret key. The viewing key can be given to third parties for accounting/auditing purposes or to help the authorities track down illegal activity.
+
+* The **proof authorisation key** grants the right to generate ZKPs for transfer validity on behalf of the holder of the corresponding secret key. This can be useful to delegate proof generation to specialised hardware without granting spending rights.
+
+* The **secret key**, also known as **spending key**, grants the user full rights over the assets owned by it, that is, those zkAssets sent to the corresponding zkAddress and not yet spent.
+
+## zkAddress System: the math
+
+It would be disastrous if someone could deduce the spending key or the viewing key from the zkAddress. Let's see why the zkAddress system is cryptographically secure.
+
+The spending key is a field element, and the proof authorisation key is an elliptic curve point derived from the spending key by multiplication by a generator. The same applies to the viewing key and the zkAddress. These two derivations are secure because of the discrete logarithm assumption. 
```

Comment:
```suggestion
The spending key is a field element, and the proof authorization key is an elliptic curve point derived from the spending key by multiplication by a generator. The same applies to the viewing key and the zkAddress. These two derivations are secure because of the discrete logarithm assumption. 
```